js/views/chatHeadsVw.js

```diff
@@ -18,6 +18,18 @@ module.exports = baseVw.extend({
         this.createChatHead(md).render().el
       );
     });
+
+    this.$chatHeadsContainer = $('.chatConversationHeads');
+    this.$headContainer = $('<div />');
+
+    this.showPerScroll = 10;
+    this.nextChatToShow = 0;
+
+    //listen to scrolling on container
+    this.scrollHandler = __.bind(
+        __.throttle(this.onScroll, 100), this
+    );
+    this.$chatHeadsContainer.on('scroll', this.scrollHandler);
```

Comment:
Sounds good.

The base view's remove will remove any child views you registered with `registerChild`. It has no idea though that you bound a scroll handler to some element oustide of the view.